newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/ExternalEventsAutoComplete.js

```diff
@@ -8,8 +9,11 @@ import SemiControlledAutoComplete, {
   type DataSource,
 } from '../../../../UI/SemiControlledAutoComplete';
 
-const getList = (project: ?gdProject): DataSource => {
-  if (!project) {
+const getList = (
+  currentSceneName: ?string,
+  project: ?gdProject
+): DataSource => {
+  if (!project || !currentSceneName) {
```

Comment:
I don't think we should return an empty array if the currentSceneName is null/undefined/falsy? 
In events based function, the currentSceneName will be undefined, and still we should autocomplete with the layouts and external events?

Reply:
returning an empty array is not a problem i think

Reply:
Can you give an example showing that this is ok? :)

Reply:
Hey ! When there is no current layout, you are inside an extension, and there link events do not work anyways ([see here](https://github.com/4ian/GDevelop/blob/36fb4ec9b2cf5529ddf621be8a0a0e84f8572f37/GDJS/GDJS/Extensions/Builtin/CommonInstructionsExtension.cpp#L54)), so no autocompletions should be shown

Reply:
Ok thanks!